server/test/Tests/Search.spec.js

```diff
@@ -40,17 +40,17 @@ describe('The search API', function () {
       .set({ Authorization: superAdminToken })
       .end((err, res) => {
         expect(res.status).to.equal(200);
-        expect(res.body.length).to.equal(2);
+        expect(res.body.users.length).to.equal(2);
```

Comment:
This is a happy test. loop through the content of the results of `res.body.users`